tests/scripts/custodian/regions.txt

```diff
@@ -0,0 +1,4 @@
+us-east-1
```

Comment:
Regions can be hardcoded since we would want to operate on all regions. Longer term we also want to work in Azure and GCP so we may want to do the same for those rather than having many files.

Reply:
removed, no longer a file and is hard coded in the dockerfile